asterixdb/asterix-doc/pom.xml

```
@@ -45,6 +45,13 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-antrun-plugin</artifactId>
+        <dependencies>
+          <dependency>
+            <groupId>org.apache.asterix</groupId>
+            <artifactId>asterix-lang-sqlpp</artifactId>
+            <version>0.9.6-SNAPSHOT</version>
```

can this be ${parent.version} ? We always keep this module's versison the same as the parent's. If it's like this it would complicate the version increment process during releases.

---

and by "this" I mean <version>, sorry. I thought that comment would get inserted right at line 52.

---

I'm actually not sure how you do that, I've never really worked with Maven before, but do you mean like that?
```suggestion
            <version>${parent.version}</version>
```